### PR TITLE
Client Upgrade Scenario rework

### PR DESCRIPTION
**Pull request by jyejare**
Changes made:
1. Some rework and fixing was required to from client scenarios pov.
2. Some changes are made wrt to initializing configurations that can be utilized in all scenario tests in file ```upgrade_tests/test_scenarios/__init__.py```
3. functions in docker client creation changed to use different org

### CODE REVIEW

**Comment by jyejare** [comment]
I am on PTO and will address comments post PTO

File: upgrade/helpers/docker.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 def generate_satellite_docker_clients_on_rhevm(
-        client_os, clients_count, custom_ak=None):
+        client_os, clients_count, custom_ak=None, org_label=None):
```
Comment (san7ket) [inline upgrade/helpers/docker.py]: doc string for org_label param missing

File: upgrade_tests/test_scenarios/test_client.py [hunk 2]
```diff
@@ -42,11 +235,12 @@ class Scenario_preupgrade_client_package_installation(TestCase):
         4. Check if the package is installed on the pre-upgrade client
     """
     docker_vm = os.environ.get('DOCKER_VM')
-    ak = os.environ.get('RHEV_CLIENT_AK_RHEL7')
-    cv = entities.ActivationKey(organization=1).search(
-        query={'search': 'name={}'.format(ak)}
-    )[0].content_view
-    package_name = 'horse'
+    org = entities.Organization(id=1).read()
+    repo_url = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
```
Comment (sghai) [inline upgrade_tests/test_scenarios/test_client.py]: is this url one time used ? may be good to define at one place and then reuse further in other tests. but not a blocker to merge.

**Comment by sghai** [comment]
ack pending a comment

**Comment by jyejare** [comment]
@sghai  and @san7ket  I worked on your comments, please have a look

**Comment by sghai** [comment]
travis is not happy. please fix it and we are good to merge.

**Comment by jyejare** [comment]
@sghai  The travis is now happy! Please merge !

**Pull request merged**